scylla/src/transport/load_balancing/default.rs

```diff
@@ -757,7 +757,7 @@ mod tests {
                 for got in gots {
                     // First, make sure that `got` has the right number of items,
                     // equal to the sum of sizes of all expected groups
-                    let combined_groups_len = self.groups.iter().map(|s| s.len()).sum();
+                    let combined_groups_len: usize = self.groups.iter().map(|s| s.len()).sum();
```

Comment:
I removed the `usize` and it still compiles with rust 1.68, but I'm a fan of having explicit type annotations so we could leave it as is.